fc/person_request.go

```diff
@@ -16,7 +16,7 @@ type PersonRequest struct {
 	PartnerId  string      `json:"partnerId,omitempty"`
 	LiNonId    string      `json:"li_nonid,omitempty"`
 	Confidence string      `json:"confidence,omitempty"`
-	Infer      bool        `json:"infer,omitempty"`
+	Infer      *bool       `json:"infer,omitempty"`
```

Comment:
Not sure why you'd need a pointer to a boolean. Should be able to pass/assign by value.

Reply:
Unfortunately, it just doesn't work in its current state when assigning by value. Looking at the tests, notice on the existing test [here](https://github.com/fullcontact/fullcontact-go/blob/c4570eadc047ed9752c42d60b062e6b2ec46d294/fc/person_request_test.go#L23) that `WithInfer(false)` is being set, but `"infer": false` is missing from the [json payload](https://github.com/fullcontact/fullcontact-go/blob/c4570eadc047ed9752c42d60b062e6b2ec46d294/fc/person_request_test.go#L16).

I attempted to outline the problem in-depth in my PR description, so I'm hoping that gives good context to the issue. Let me know if I can clarify anything to help out 👍

Reply:
Reading up a little on this, https://vsupalov.com/go-json-omitempty/  - this is an issue.  Sounds like infer defaults to true on our API side. Your approach seems reasonable.